### PR TITLE
Makes the code compile under 512.1437

### PR DESCRIPTION
**Pull request by MadmanMartian**
:cl: 
tweak: Code is now compiled under BYOND version 512.1437
/:cl:

[why]: # because it's a pain to keep reverting from beta to stable to compile and make changes, and that rogue dot was weird and annoying.

As suggested by #3574

### CODE REVIEW

File: code/game/gamemodes/wizard/soulstone.dm [hunk 1]
```diff
@@ -119,7 +119,7 @@
 ////////////////////////////Proc for moving soul in and out off stone//////////////////////////////////////
 
 
-/obj/item/device/soulstone/proc/transfer_soul(choice as text, target, mob/user).
```
Comment (Qustinnus) [inline code/game/gamemodes/wizard/soulstone.dm]: @monster860 did this do anything lol

**Comment by Qustinnus** [comment]
image
lol'd.

anyways good PR you missed a few mentions of filter in code/game/machinery/computer/apc_control.dm

**Comment by MadmanMartian** [comment]
Weird. Those errors don't show during compile my side.

**Comment by monster860** [comment]
perhaps you aren't actually running a 512 compiler?

**Comment by MadmanMartian** [comment]
proof_of_512

**Pull request merged**